rust/src/lib.rs

```diff
@@ -52,6 +52,7 @@ pub mod filemetadata;
 pub mod flowgraph;
 pub mod function;
 pub mod headless;
+pub mod linearview;
```

Comment:
Correct me if I'm wrong, or this was intentional but it appears this file needs the following line:

```pub mod interaction;```

in order to include interaction.rs as a module that can be used when including binaryninja as a cargo dependency?

Reply:
Yes, you are absolutely correct. Seems I dropped that at some point when rebasing things.